api/tokens.go

```diff
@@ -197,6 +227,16 @@ func buildCognitoRequest(authParams AuthParams, clientId string, clientSecret st
 	return authInput
 }
 
+func buildAdminSignoutRequest(authParams AuthParams, userPoolId string) (adminSignoutInput *cognitoidentityprovider.AdminUserGlobalSignOutInput) {
+
+	adminSignoutInput = &cognitoidentityprovider.AdminUserGlobalSignOutInput{
+		Username:   &authParams.Email,
+		UserPoolId: &userPoolId,
+	}
+
+	return adminSignoutInput
```

Comment:
You may as well move this into the  `adminUserGlobalSignOut` func - it would use less lines.

Reply:
Thank you